Annotato/Annotato/ViewModels/Annotation/AnnotationPaletteViewModel.swift

```diff
@@ -60,17 +70,35 @@ class AnnotationPaletteViewModel: ObservableObject {
     func enterEditMode() {
         isEditing = true
         parentViewModel?.enterEditMode()
+        enterMaximizedMode()
     }
 
     func enterViewMode() {
         isEditing = false
         parentViewModel?.enterViewMode()
+
+        guard isMinimizedByUser else {
+            return
+        }
+
+        enterMinimizedMode()
```

Comment:
Do we need this? when entering view mode, we just keep whatever state it was right? If it was already minimized it remains minimized and if it was already maximized it remains maximized?